tests/integration/query/commits/branchables/peer_test.go

```diff
@@ -50,76 +48,153 @@ func TestQueryCommitsBranchables_SyncsAcrossPeerConnection(t *testing.T) {
 			},
 			testUtils.WaitForSync{},
 			testUtils.Request{
-				NodeID: immutable.Some(0),
 				Request: `query {
-					commits {
-						cid
-						links {
+						commits {
 							cid
+							links {
+								cid
+							}
 						}
-					}
-				}`,
+					}`,
 				Results: map[string]any{
 					"commits": []map[string]any{
 						{
-							"cid": testUtils.NewUniqueCid("collection"),
+							"cid": testUtils.NewUniqueCid(0),
```

Comment:
note: I was playing around here using ints as cid ids and forgot it was an existing test - I still prefer strings, but let me know your thoughts :)

Reply:
Is the input param needed or could the function generate a random ID in the `NewUniqueCid` func?

Reply:
The input param is purely descriptive, it has no relation to the value of the cid that it represents

Reply:
Although, if I misunderstood you - yes there needs to be a value as otherwise there is no way to describe which values should/shouldn't be the same

Reply:
I ask because if generated internally the pattern for reuse could be:

```go
cidA := testUtils.NewUniqueCid()
cidB := testUtils.NewUniqueCid()
```

Just a thought I had, no need to change the implementation.

Reply:
Ah, I guess we could just make it a pointer and check for equality of the reference - would be a better non-descriptive version than using `int`, but a fair bit more wordy as you'd need to declare the variables if used in multiple places.  (I still prefer strings atm though)

Reply:
The strings definitely make this more comprehensible. I much prefer how it was before.

Reply:
I'll change this test back to strings in a sec, I can revert if anyone comes back very soon before merge in favour of the ints @islamaliev  

- [ ] Use strings in test